modiscolite/util.py

```diff
@@ -2,9 +2,27 @@
 # Authors: Jacob Schreiber <jmschreiber91@gmail.com>
 # adapted from code written by Avanti Shrikumar 
 
+from enum import Enum
 import numpy as np
 from numba import njit
 
+
+class MemeDataType(Enum):
```

Comment:
Personally, I think we should remove this and just explicitly spell out the options in the command-line tool. I understand why having this object is reasonable software engineering but I prefer to have as few objects and variables as possible so that the code is as easy to pick up by someone new as possible

Reply:
Fair